### ISSUE TITLE
Crash on profile select

### ISSUE THREAD

**Issue by kiasoc5** [igo95862/bubblejail#17]
OS: Arch Linux
Versions: 0.5.0 and latest git
How to reproduce:
1. Run bubblejail-config
2. Create instance
3. Select any instance
Then it crashes.
```
Traceback (most recent call last):
  File "/usr/lib/bubblejail/python_packages/bubblejail/bubblejail_gui_qt.py", line 434, in selection_changed
    self.current_profile = BubblejailDirectories.profile_get(new_text)
  File "/usr/lib/bubblejail/python_packages/bubblejail/bubblejail_directories.py", line 109, in profile_get
    with open(possible_profile_path, mode='b') as profile_file:
ValueError: Must have exactly one of create/read/write/append mode and at most one plus
```

**Comment by igo95862** [comment]
Very trivial error on my part.

Should be fixed in new commit. Can you give it a test?

I need to set up better testing CI.

**Comment by igo95862** [comment]
I also released 0.5.1 with the fix.

**Comment by kiasoc5** [comment]
Thanks it works.
